apps/assessment_registry/schema.py

```diff
@@ -0,0 +1,122 @@
+
+
+class AssessmentRegistryType(
+        DjangoObjectType,
+        UserResourceMixin,
+        ClientIdMixin,
+):
+
+    class Meta:
+        model = AssessmentRegistry
+        fields = (
+            "id", "lead", "project", "lead_group", "bg_countries", "bg_crisis_start_date",
+            "cost_estimates_usd", "no_of_pages", "data_collection_start_date", "data_collection_end_date",
+            "publication_date", "lead_organizations", "international_partners", "donors", "national_partners",
+            "governments", "objectives", "data_collection_techniques", "sampling", "limitations", "locations",
+        )
+
+    bg_crisis_type = graphene.Field(AssessmentRegistryCrisisTypeEnum, required=True)
+    bg_crisis_type_display = EnumDescription(source='get_bg_crisis_type_display', required=True)
+    bg_preparedness = graphene.Field(AssessmentRegistryPreparednessTypeEnum, required=True)
+    bg_preparedness_display = EnumDescription(source='get_bg_preparedness_display', required=True)
+    external_support = graphene.Field(AssessmentRegistryExternalSupportTypeEnum, required=True)
+    external_support_display = EnumDescription(source='get_external_support_display', required=True)
+    coordinated_joint = graphene.Field(AssessmentRegistryCoordinationTypeEnum, required=True)
+    coordinated_joint_display = EnumDescription(source='get_coordinated_joint_display', required=True)
+    details_type = graphene.Field(AssessmentRegistryDetailTypeEnum, required=True)
+    details_type_display = EnumDescription(source='get_details_type_display', required=True)
+    family = graphene.Field(AssessmentRegistryFamilyTypeEnum, required=True)
+    family_display = EnumDescription(source='get_family_display', required=True)
+    frequency = graphene.Field(AssessmentRegistryFrequencyTypeEnum, required=True)
+    frequency_display = EnumDescription(source='get_frequency_display', required=True)
+    confidentiality = graphene.Field(AssessmentRegistryConfidentialityTypeEnum, required=True)
+    confidentiality_display = EnumDescription(source='get_confidentiality_display', required=True)
+    language = graphene.List(graphene.NonNull(AssessmentRegistryLanguageTypeEnum), required=True)
+    focuses = graphene.List(graphene.NonNull(AssessmentRegistryFocusTypeEnum), required=True)
+    sectors = graphene.List(graphene.NonNull(AssessmentRegistrySectorTypeEnum), required=True)
+    protection_info_mgmts = graphene.List(graphene.NonNull(AssessmentRegistryProtectionInfoTypeEnum), required=True)
+    affected_groups = graphene.Field(AssessmentRegistryAffectedGroupTypeEnum, required=True)
+    affected_groups_display = EnumDescription(source='get_affected_groups_display', required=True)
+    methodology_attributes = graphene.List(graphene.NonNull(MethodologyAttributeType), required=False)
+    additional_documents = graphene.List(graphene.NonNull(AdditionalDocumentType), required=False)
+
```

Comment:
We will need to define `get_custom_queryset` to filter by active projects. Use lead schema as reference.